dScripts/ActivityManager.cpp

```diff
@@ -124,8 +124,6 @@ void ActivityManager::ActivityTimerStart(Entity* self, const std::string& timerN
 	auto* timer = new ActivityTimer{ timerName, updateInterval, stopTime };
 	activeTimers.push_back(timer);
 
-	Game::logger->Log("ActivityManager", "Starting timer '%s', %f, %f", timerName.c_str(), updateInterval, stopTime);
```

Comment:
Removing spammy logs since I'm testing with AGS, as it has a bunch of paths that are meant to bring the enemies into the map towards the players, this will make AGS harder :smiling_imp: